tests/test_data/test_neon_benthic_data_translator.py

```diff
@@ -130,52 +128,36 @@
     ),
 }
 
+def neon_envo_mappings_file():
```

Comment:
Addressing your comment about making live network requests in tests: https://github.com/microbiomedata/nmdc-runtime/pull/308#discussion_r1416066358

I've added mock data to the test files directly.